.github/workflows/release.yml

```diff
@@ -386,8 +431,6 @@ jobs:
           - amazonlinux:2023
           - registry.access.redhat.com/ubi8/ubi
           - registry.access.redhat.com/ubi9/ubi
-          - amazonlinux:2
-          - registry.access.redhat.com/ubi7/ubi
```

Comment:
If we are shipping packages, we should test them. Perhaps the template conditional is incorrect?

Reply:
added again